### PR TITLE
Added support for larger on-mob and item sprites.

### PR DESCRIPTION
**Pull request by Generalcamo**
Added backend support for larger on-mob and item sprites, such as 64x64 sprites:

image

Credit: https://github.com/cmss13-devs/cmss13/pull/2452

### CODE REVIEW

**Comment by Generalcamo** [comment]
!review

**Review by SleepyGemmy — APPROVED** [review]

File: code/_helpers/icons.dm [hunk 1]
```diff
@@ -1224,3 +1224,39 @@ lighting determines lighting capturing (optional), suppress_errors suppreses err
 
 	var/icon/I = getFlatIcon(thing)
 	return icon2html(I, target, sourceonly = sourceonly)
+
+/**
+ * Centers an image.
+ * Requires:
```
Comment (FluffyGhoster) [inline code/_helpers/icons.dm:1230]: Single newlines in DMdocs are not represented in the mouse over tooltip in VSC, you need an empty DMdoc line to go newline

File: code/game/objects/items.dm [hunk 2]
```diff
@@ -18,6 +18,11 @@
 
 	var/storage_cost
 
+	///Dimensions of the icon file used when this item is worn, eg: hats.dmi (32x32 sprite, 64x64 sprite, etc.). Allows inhands/worn sprites to be of any size, but still centered on a mob properly
+	var/worn_x_dimension = 32
+	///Dimensions of the icon file used when this item is worn, eg: hats.dmi (32x32 sprite, 64x64 sprite, etc.). Allows inhands/worn sprites to be of any size, but still centered on a mob properly
```
Comment (FluffyGhoster) [inline code/game/objects/items.dm:23]: Separate the dmdoc with variable from the other dmdoc with variable

**Review by FluffyGhoster — APPROVED** [review]

File: code/_helpers/icons.dm (hunk 1 above)
Comment (Generalcamo) [reply code/_helpers/icons.dm:1230]: Ported 1:1 from /tg/, will fix

**Pull request merged**